ui/src/app/utils/index.tsx

```diff
@@ -531,12 +531,12 @@ export function highlightSearchTerm(searchTerm: string, stringToHighlight: strin
 
 // render a float value as US currency, rounded to cents: 255.372793 -> $255.37
 // negative values are rendered as $0
-export function renderUSD(value: number) {
+export function renderUSD(value: number): string {
```

Comment:
`render` implies a side effect. I'd call this `formatCreditUsage`. (The negative to zero conversion indicates to me that maybe we shouldn't put it in a public utils file, as that's not standard, and would be an easy way to generate bugs.)

Is there a specific requirement to drop the cents when it's zero? Why not when it's `$1`? Or can we just round to the nearest dollar?

Reply:
Renamed away from "render" and with a name to indicate that this is specifically for free credit values (usage, balance, quota/override).

The requirement is to use the string "$0" for negative values.